infra/helm/meshdb/templates/meshweb.yaml

```diff
@@ -47,18 +47,38 @@ spec:
               mountPath: /opt/meshdb/static
           {{ if eq .Values.meshweb.liveness_probe "true" }}
           livenessProbe:
-           exec:
-             command:
-               - curl
-               - http://127.0.0.1:{{ .Values.meshweb.port }}/api/v1
-           periodSeconds: 3
-           initialDelaySeconds: 4
-           timeoutSeconds: 3
-           {{ end }}
-          livenessProbe:
-            {{- toYaml .Values.livenessProbe | nindent 12 }}
+            exec:
+              command:
+                - bash
+                - -c
+                - 'curl http://127.0.0.1:{{ .Values.meshweb.port }}/api/v1/ -H "Host: db.nycmesh.net" -s | grep meshin'
```

Comment:
The grep is fine but knowing that it's `we're meshin'` confused my brain for a sec lol